flatland_server/src/flatland_server_node.cpp

```diff
@@ -90,7 +90,7 @@ class FlatlandServerNode : public rclcpp::Node
       }
       get_parameter_or("update_rate", update_rate_, 200.0f);
       get_parameter_or("step_size", step_size_, 1.0f/200.0f);
-      get_parameter_or("show_viz", show_viz_, 0.0f);
+      get_parameter_or("show_viz", show_viz_, false);
```

Comment:
This was changed from bool (master/ros1) to double (@josephduchesne ros2 commits).
Is there a reason to save it as double? I changed it back to bool.